sunpy/lightcurve/tests/test_lightcurve.py

```diff
@@ -39,6 +40,10 @@ def test_input(data, index):
     assert lc.data.index[0] == base
     assert lc.data.index[-1] == base - datetime.timedelta(minutes=24 * 60 - 1)
 
+def test_time_range():
+    time_range = TimeRange('2010/03/04 00:10', '2010/03/04 00:20')
+    assert isinstance(time_range, sunpy.time.TimeRange)
+
```

Comment:
This test is not testing anything in `lightcurve` but in Timerange itself, which it's already being tested in [`test_timerange.py`](https://github.com/sunpy/sunpy/blob/master/sunpy/time/tests/test_timerange.py#L25)